File: Python/cadastroMusicas.py
musicas = []

while True:

    print("\nMenu")
    print("1 - Adicionar música")
    print("2 - Remover música")
    print("3 - Listar músicas")
    print("4 - Sair")

    opcao = input("Escolha: ")

    if opcao == "1":
        musica = input("Nome da música: ")
        musicas.append(musica)

    elif opcao == "2":
        musica = input("Nome da música para remover: ")
        if musica in musicas:
            musicas.remove(musica)
            print("Música removida!")
        else:
            print("Música não encontrada")

    elif opcao == "3":
        print("\nLista de músicas:")
        for m in musicas:
            print("-", m)

    elif opcao == "4":
        break

File: Java/CadastroMusicas.java
import java.util.ArrayList;
import java.util.Scanner;

public class CadastroMusicas {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        ArrayList<String> musicas = new ArrayList<>();

        while(true){

            System.out.println("\nMenu");
            System.out.println("1 - Adicionar música");
            System.out.println("2 - Remover música");
            System.out.println("3 - Listar músicas");
            System.out.println("4 - Sair");

            int opcao = sc.nextInt();
            sc.nextLine();

            if(opcao == 1){

                System.out.print("Nome da música: ");
                String musica = sc.nextLine();

                musicas.add(musica);

            }

            else if(opcao == 2){

                System.out.print("Nome da música para remover: ");
                String musica = sc.nextLine();

                if(musicas.remove(musica)){
                    System.out.println("Música removida!");
                }else{
                    System.out.println("Música não encontrada");
                }

            }

            else if(opcao == 3){

                System.out.println("\nLista de músicas:");

                for(String m : musicas){
                    System.out.println("- " + m);
                }

            }

            else if(opcao == 4){
                break;
            }

        }

        sc.close();
    }
}